test/Microsoft.AspNet.Razor.Test/Parser/CSharp/CSharpRazorCommentsTest.cs

```diff
@@ -173,5 +176,36 @@ public void UnterminatedRazorCommentInVerbatimBlock()
                 new RazorError(RazorResources.ParseError_RazorComment_Not_Terminated, 2, 0, 2),
                 new RazorError(RazorResources.FormatParseError_Expected_EndOfBlock_Before_EOF(RazorResources.BlockName_Code, "}", "{"), 1, 0, 1));
         }
+
+        [Fact]
+        public void RazorCommentInMarkup()
+        {
+            ParseDocumentTest(
+                "<p>" + Environment.NewLine
+                + "@**@" + Environment.NewLine
+                + "</p>",
```

Comment:
Hmm, this matches @NTaylorMullen's [earlier](https://github.com/aspnet/Razor/pull/482#issuecomment-130463445) example:
``` html
<p>
  @* Hello *@    @* World *@
</p>
```